### PR TITLE
FIX: DeviceQuickActions getEnabled

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

File: components/DeviceQuickActions.tsx [hunk 1]
```diff
@@ -1,21 +1,39 @@
...
 
 const DeviceQuickActionsStorageKey = 'DeviceQuickActionsEnabled';
 
-function DeviceQuickActions(): JSX.Element | null {
+export async function setEnabled(enabled: boolean = true): Promise<void> {
+  await AsyncStorage.setItem(DeviceQuickActionsStorageKey, JSON.stringify(enabled));
+}
+
+export async function getEnabled(): Promise<boolean> {
+  console.warn('getEnabled22');
```
Comment (Overtorment) [inline components/DeviceQuickActions.tsx:21]: Debug log

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/iqkfkzhvkt65xq7kuranedex54?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1420)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/rekbo57waepvwkt5dksbmlzwre?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1424)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**